ext/project_data.yaml

```diff
@@ -0,0 +1 @@
+build_defaults.yaml
```

Comment:
this should be a symlink to ext/build_defaults.yaml but I have no idea who uses it

Reply:
it is
i guess this is how git shows it
```
~/Workspace/pxp-agent-vanagon  PA-3336                                                                                           ruby-2.6.3
❯ ls -l ext
total 8
build_defaults.yaml
project_data.yaml -> build_defaults.yaml
```